spec/atom-portable-spec.coffee

```diff
@@ -0,0 +1,71 @@
+path = require "path"
+fs = require 'fs-plus'
+temp = require "temp"
+rimraf = require "rimraf"
+AtomPortable = require "../src/browser/atom-portable"
+
+describe "Set Portable Mode", ->
+  portableAtomHomePath = path.join(path.dirname(process.execPath), "../.atom").toString()
```

Comment:
`path.join` always returns a string so I don't think you need the `toString` here.